.dockerignore

```diff
@@ -0,0 +1,35 @@
+node_modules
```

Comment:
Why are these changes in here?

Reply:
Somehow GitHub can't divide the merged changes from the ones done on the branch itself.